tests/framework/OutStreams/tests

```diff
@@ -58,4 +58,9 @@
   input = 'test_legend.xml'
   output = 'plot/legendTest.png'
  [../]
+ [./line]
+  type = 'RavenFramework'
+  input = 'line_plot.xml'
+  output = 'linePlot/to_plot_line.png'
```

Comment:
can you add this as "image" test? at least when we activate it we will have it ready.

Reply:
added